code/datums/materials/basemats.dm

```diff
@@ -197,7 +197,7 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	color = "#bb8e53"
 	strength_modifier = 0.5
 	categories = list(MAT_CATEGORY_RIGID = TRUE)
-	value_per_unit = 0.06
+	value_per_unit = 0.01
```

Comment:
I don't really mind you doing this change but do keep in mind you need to document these changes in the changelog

Reply:
My B, Q